simpleTest1/appFirstPb.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (array) {
+
+    var sum = 0;
+    let notNumber = 0;
+
+    if(array.length > 0) {
```

Comment:
Do you really need this here, what will happen if you don't have it and the array passed is `[]`?

Reply:
If I don't have it, the sum will be 0, so yes I do not need that condition there.